lib/ain-grpc/src/rpc/web3.rs

```diff
@@ -0,0 +1,33 @@
+use ain_evm::bytes::Bytes;
+use digest::Digest;
+use jsonrpsee::core::RpcResult;
+use jsonrpsee::proc_macros::rpc;
+use primitive_types::H256;
+
+#[rpc(server, client, namespace = "web3")]
+pub trait MetachainWeb3RPC {
+    /// Returns the current network ID as a string.
+    #[method(name = "clientVersion")]
+    fn client_version(&self) -> RpcResult<String>;
+    /// Returns the current network ID as a string.
+    #[method(name = "sha3")]
+    fn sha3(&self, input: Bytes) -> RpcResult<H256>;
+}
+
+#[derive(Default)]
+pub struct MetachainWeb3RPCModule {}
+
+impl MetachainWeb3RPCServer for MetachainWeb3RPCModule {
+    fn client_version(&self) -> RpcResult<String> {
+        let version = env!("CARGO_PKG_VERSION");
+        let commit = option_env!("GIT_HASH").unwrap_or("unknown");
+        let os = std::env::consts::OS;
+
+        Ok(format!("Metachain/v{}/{}-{}", version, os, commit))
```

Comment:
Let's pull this from config.h, so it's consistent, and make config.h itself to add what's needed.

Reply:
https://github.com/DeFiCh/ain/pull/2482

@prasannavl hope this PR does so